source/routing/raptor_api.cpp

```diff
@@ -1260,7 +1262,7 @@ bool way_later(const Journey & j1, const Journey & j2,
 
     auto get_pseudo_duration = [&](const Journey & j) {
         auto dt = clockwise ? j.arrival_dt : j.departure_dt;
-        return std::abs((double)dt - requested_dt);
+        return std::abs(double(dt) - double(requested_dt));
```

Comment:
My bad !

Reply:
@nonifier Generally speaking, c style cast is bad in c++

Reply:
In this case, we are trying to convert a built-in type implicitly convertible to another one. 
So there is no much risk of doing it with a C-style cast as the c++ compiler will do a `static_cast` by definition anyway (after a `const_cast` that will be ignored as there is no constness). (see [explicit cast](https://en.cppreference.com/w/cpp/language/explicit_cast)).

Nonetheless, I've been lazy casting only one parameter and I'm finding excuses to slack off :D